tests/src/404.js

```diff
@@ -1,4 +1,5 @@
 const puppeteer = require('puppeteer');
+const axios = require('axios');
```

Comment:
Since we use node22, refactor with `fetch`. It could prevent adding a new dependency :)

Reply:
Done, thanks for the tip :)